packages/website/src/features/Debug/DebugPage.tsx

```diff
@@ -28,93 +36,111 @@ export const DebugPage = () => {
           as="h2"
           size="lg"
           fontWeight={600}
+          letterSpacing="0.2px"
+          mb={2.5}
+        >
+          Debug a Protocol
+        </Heading>
+        <Text
           pb={4}
           mb={4}
           borderBottom="1px solid"
           borderBottomColor="gray.800"
-          letterSpacing="0.2px"
+          fontSize="xl"
+          color="gray.400"
         >
-          Debugging your Contracts with Cannon: What to do when things go wrong
-        </Heading>
-        <Text mb={4}>
-          We have all been there. Our production app is emitting a strange hex
-          error, but our wallet wasn’t able to explain the actual cause. Worse,
-          the error could be the dreaded `0x` (aka an error with no revert
-          reason)! Or perhaps you have a transaction on gnosis safe, but its
-          getting sent cross chain or through a proxy of some sort, so the
-          function call cant be verified because its obscured.
+          Troubleshoot issues with your protocol during development and testing.
```

Comment:
Troubleshoot in all stages--development, testing, *and* production (debugging doesn't stop jusst because app is in prod). Important to highlight the process is basically the same for every app stage.